src/gui/options.ui

```diff
@@ -566,6 +566,57 @@
                     </property>
                    </widget>
                   </item>
+		  <item row="5" column="0">
```

Comment:
Starting from here is all messed up (saw it from qt designer).

You should **not** put the label & spinbox inside of QGroupBox.
Actually let me fixing it is much faster. Use the following diff to patch your commit
``` diff
diff --git a/src/gui/options.ui b/src/gui/options.ui
index 52aa5cd..9154079 100644
--- a/src/gui/options.ui
+++ b/src/gui/options.ui
@@ -521,7 +521,7 @@
              <x>0</x>
              <y>0</y>
              <width>487</width>
-             <height>965</height>
+             <height>997</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
@@ -555,8 +555,8 @@
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
-                 <layout class="QGridLayout" name="gridLayout_15">
-                  <item row="0" column="0">
+                 <layout class="QVBoxLayout" name="verticalLayout_11">
+                  <item>
                    <widget class="QCheckBox" name="checkAdditionDialogFront">
                     <property name="text">
                      <string>Bring torrent dialog to the front</string>
@@ -566,56 +566,42 @@
                     </property>
                    </widget>
                   </item>
-		  <item row="5" column="0">
-                   <widget class="QGroupBox" name="groupBox_5">
-                    <property name="title">
-                     <string/>
-                    </property>
-                    <widget class="QSpinBox" name="spinSaveHistoryListSize">
-                     <property name="geometry">
-                      <rect>
-                       <x>290</x>
-                       <y>0</y>
-                       <width>62</width>
-                       <height>31</height>
-                      </rect>
-                     </property>
-                     <property name="sizePolicy">
-                      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
-                       <horstretch>0</horstretch>
-                       <verstretch>0</verstretch>
-                      </sizepolicy>
-                     </property>
-                     <property name="minimum">
-                      <number>1</number>
-                     </property>
-                     <property name="maximum">
-                      <number>100</number>
-                     </property>
-                     <property name="value">
-                      <number>10</number>
-                     </property>
-                    </widget>
-                    <widget class="QLabel" name="label_4">
-                     <property name="geometry">
-                      <rect>
-                       <x>10</x>
-                       <y>-40</y>
-                       <width>321</width>
-                       <height>108</height>
-                      </rect>
-                     </property>
-                     <property name="sizePolicy">
-                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
-                       <horstretch>0</horstretch>
-                       <verstretch>0</verstretch>
-                      </sizepolicy>
-                     </property>
-                     <property name="text">
-                      <string>Size of history for saved locations:</string>
-                     </property>
-                    </widget>
-                   </widget>
+                  <item>
+                   <layout class="QHBoxLayout" name="horizontalLayout">
+                    <item>
+                     <widget class="QLabel" name="label_4">
+                      <property name="text">
+                       <string>Size of history for saved locations:</string>
+                      </property>
+                     </widget>
+                    </item>
+                    <item>
+                     <widget class="QSpinBox" name="spinSaveHistoryListSize">
+                      <property name="minimum">
+                       <number>1</number>
+                      </property>
+                      <property name="maximum">
+                       <number>100</number>
+                      </property>
+                      <property name="value">
+                       <number>10</number>
+                      </property>
+                     </widget>
+                    </item>
+                    <item>
+                     <spacer name="horizontalSpacer_10">
+                      <property name="orientation">
+                       <enum>Qt::Horizontal</enum>
+                      </property>
+                      <property name="sizeHint" stdset="0">
+                       <size>
+                        <width>0</width>
+                        <height>0</height>
+                       </size>
+                      </property>
+                     </spacer>
+                    </item>
+                   </layout>
                   </item>
                  </layout>
                 </widget>
@@ -1087,7 +1073,7 @@
             <rect>
              <x>0</x>
              <y>0</y>
-             <width>480</width>
+             <width>450</width>
              <height>658</height>
             </rect>
            </property>
@@ -1617,8 +1603,8 @@
             <rect>
              <x>0</x>
              <y>0</y>
-             <width>502</width>
-             <height>538</height>
+             <width>376</width>
+             <height>444</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
@@ -2750,8 +2736,8 @@
             <rect>
              <x>0</x>
              <y>0</y>
-             <width>98</width>
-             <height>28</height>
+             <width>493</width>
+             <height>538</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_36">
```